### DISCUSSION TITLE
w64devkit virus or not?

### DISCUSSION THREAD

**Discussion by zhanghui-china** [ggml-org/llama.cpp#3464] [Q&A] [answered]
i want to compile code on Windows11,
when i use w64devkit downloaded from https://github.com/skeeto/w64devkit/releases
it shows 
![e7a27fab4d378f0fe4e10cd7f6772f5](https://github.com/ggerganov/llama.cpp/assets/63148804/6951de9b-5140-4c95-b91a-6c1def3d601e)

so i cannot finish compiling llama.cpp
![a27b869508566cddd2a2c20e4f2c093](https://github.com/ggerganov/llama.cpp/assets/63148804/9aedbf75-5004-41aa-934a-ab2e36f7a8cd)

what can i do ?

**Comment by KerfuffleV2** [comment, marked as answer]
That project looks active/reputable. This is almost certainly just a false positive. As part of a compiler, `collect2.exe` will be manipulating executable files which is the kind of thing that looks suspicious to virus checkers. Basically, you'll _probably_ be fine just telling your virus checker to trust it.

Of course, I can't guarantee anything so it's up to you whether you want to trust it or not.

**Comment by rikoras** [comment]
想做开发的话还是不要用360了~

**Reply by zhanghui-china** [reply]
3q。

**Comment by mtmcp** [comment]
For what it’s worth, the same thing happened to me when I compiled on MSYS2—it quarantined ’cmake’ when I ran the generate command.

You can submit the binaries to VirusTotal (https://www.virustotal.com/gui/home/upload) to see the scan results. In the case of cmake it was flagged for making a legitimate remote request to a certificate authority CDN.
